### PR TITLE
Incremental changes to navbar component

### PR DESCRIPTION
**Pull request by Chrischuck**
#399 
Screenshot 2023-01-28 at 12 57 57 PM

### CODE REVIEW

**Review by jtklein — COMMENTED** [review]
I have not looked at everything in detail yet. But I can send you more about the design specs if you want to continue working on this:
The outermost two icons are both 32x32 px.
The two elements around the center button are both 40x40px
And the center AddObsButton is 69x69px.

**Comment by jtklein** [comment]
@Chrischuck we have now included our own icons in a font file. Could you update this PR from main and resolve the conflicts?

File: src/components/SharedComponents/NavBar/NavButton.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  size: number,
+};
+
+const imgStyle = ( size, active ) => StyleSheet.create( {
+  img: {
+    borderWidth: active ? 3 : 0,
+    height: size,
+    width: size,
+    borderColor: colors.inatGreen
+  }
```
Comment (Chrischuck) [inline src/components/SharedComponents/NavBar/NavButton.js:26]: I'd rather have used tailwind here...

Unfortunately setting the border width using class `border-[3px]` didn't work on android

setting height via classes didnt work on either platform

and you can't interpolate class's in tailwind

**Comment by jtklein** [comment]
Thank you @Chrischuck! I did add some more changes on top. I have removed all of your additions to testing the unrelated screens for this PR. To keep them more granular as unit tests I have added a mock for the route instead of adding a layer of elements in the testing tree. The navbar test you added is also interacting with the drawer, so it is more an integration test than unit test. I have moved it. So, probably good to add a unit test at one point.

**Pull request merged**